charts/esphome/0.0.1/app-readme.md

```diff
@@ -0,0 +1,3 @@
+# ESPHome
+
+ESPHome chart is a chart designed to deploy ESPHome in a TrueNAS SCALE kubernetes cluster.
```

Comment:
Just a thought for the future about the app-readme in general

I believe this is the app info pop up text -- Should we at some point consider using the description from charts.yaml in this file instead?

Reply:
I'm currently mostly leaving out readme's and i'm completely not-happy with any of the Chart.yaml files.

In the future I want to do one big push, standardising all readme's and Chart.yaml files... but for now, i'll leave it.
I'm also not 100% sure where this shows up, I expect it to show up in the description one day or today, yes :)

Reply:
Far as I can, its the popup bubble when you mouse over the info icon

![image](https://user-images.githubusercontent.com/35702532/107275095-4b9c5880-6a1f-11eb-8ea1-e95e0329836c.png)

Reply:
Nice, we really need to look into a nice structure for all those things later on :D